hphp/test/slow/include/fast-stat-cache.php.expect

```diff
@@ -0,0 +1 @@
+OldOldNew
```

Comment:
All right, I will add some documentation.

Yes, it helps consistency. And we add a clear cache command in Admin Server, after we deploy new files, we can call this command at once, so the cache will update as soon as possible, without the disadvantage of delay.